src/mono/msbuild/android/build/AndroidBuild.props

```diff
@@ -23,11 +26,19 @@
         _AndroidGenerateAppBundle;
         _AfterAndroidBuild
     </AndroidBuildDependsOn>
+    <AndroidBuildDependsOn Condition="'$(UseMonoRuntime)' == 'false'">
+        _InitializeCommonProperties;
+        _BeforeAndroidBuild;
+        _AndroidResolveReferences;
+        _AndroidGenerateAppBundle;
+        _AfterAndroidBuild
+    </AndroidBuildDependsOn>
 
     <!-- When building on Helix $(_CommonTargetsDir) will be properly set, otherwise we have to set it to a in-tree location -->
     <_CommonTargetsDir Condition="'$(_CommonTargetsDir)' == ''">$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', '..', 'common'))</_CommonTargetsDir>
   </PropertyGroup>
 
-  <Import Condition="'$(UseNativeAOTRuntime)' != 'true'" Project="$(_CommonTargetsDir)CommonMobileBuild.props" />
-  <Import Condition="'$(UseNativeAOTRuntime)' != 'true'" Project="$(_CommonTargetsDir)RuntimeComponentManifest.targets" />
+  <Import Condition="'$(UseNativeAOTRuntime)' != 'true' and '$(UseMonoRuntime)' == 'true'" Project="$(_CommonTargetsDir)CommonMobileBuild.props" />
```

Comment:
Out of curiosity, why we don't need `CommonMobileBuild.props` (or variant of) for CoreCLR?

Reply:
`CommonMobileBuild.props` includes only mono-specific bits. 
We could eventually rename it to avoid confusion, but it is only shipped as part of the Mono's library builder which is again an *experimental* feature.